android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java

```diff
@@ -77,6 +82,50 @@ public void show(final ReadableMap options, final Promise promise) {
     currentActivity.startActivityForResult(dropInRequest.getIntent(currentActivity), DROP_IN_REQUEST);
   }
 
+  @ReactMethod
+  public void getLastUsedPaymentMethod(final ReadableMap options, final Promise promise) {
+
+      if (!options.hasKey("clientToken")) {
+          promise.reject("NO_CLIENT_TOKEN", "You must provide a client token");
+          return;
+      }
+      Activity activity = getCurrentActivity();
+      DropInResult.fetchDropInResult((AppCompatActivity) activity, options.getString("clientToken"), new DropInResult.DropInResultListener() {
+          @Override
+          public void onError(Exception exception) {
+              // an error occurred
+              promise.reject(GET_LAST_USED_CARD_ERROR, exception.getMessage());
+          }
+
+          @Override
+          public void onResult(DropInResult result) {
+              mPromise = promise;
+              if (result.getPaymentMethodType() != null && result.getPaymentMethodNonce() != null) {
```

Comment:
can we export this as variables and then reuse it?
```suggestion
PaymentMethodType  paymentMethodType = result.getPaymentMethodType();
PaymentMethodNonce paymentMethod = result.getPaymentMethodNonce();

              if (paymentMethodType != null && paymentMethod != null) {
```

Reply:
cc @dangkhoa2708 @ducng-ehero